web/src/use/use-current-user.ts

```diff
@@ -22,6 +22,10 @@ const state = reactive<{
 export function useCurrentUser() {
   const isReady = computed(() => !state.isLoading && !state.isErrored)
 
+  const isSystemAdmin = computed(
+    () => state.currentUser?.roles?.some(({ role }) => role === RoleTypes.SYSTEM_ADMIN)
+  )
```

Comment:
`roleTypes` provides a simpler interface, and better matches the backend.
```ts
  const isSystemAdmin = computed(
    () => state.currentUser?.roleTypes.includes(RoleTypes.SYSTEM_ADMIN)
  )
```
see https://github.com/icefoganalytics/internal-data-portal/blob/0eb01fff60c6b5d72b060f89e92cf15336225531/api/src/models/user.ts#L179